### PR TITLE
Fixes return type of run_cmd when wantarray

### PR DESCRIPTION
**Pull request by b10n1k**
draft

### CODE REVIEW

File: consoles/sshVirtsh.pm [hunk 1]
```diff
@@ -631,7 +631,14 @@ sub run_cmd {
...
     delete $args{domain};
-    return $self->backend->run_ssh_cmd($cmd, %credentials, %args);
+    my @response;
+    if ($args{wantarray}) {
+        @response = $self->backend->run_ssh_cmd($cmd,  %credentials, %args);
+        return @response;
+    } else {
+        $ret = $self->backend->run_ssh_cmd($cmd, %credentials, %args);
+        return $ret
+    }
```
Comment (cfconrad) [inline consoles/sshVirtsh.pm:641]: I don't see, that your change is solving the problem. 
I would go with:

```suggestion
    if ($args{wantarray}) {
        return [$self->backend->run_ssh_cmd($cmd,  %credentials, %args)];
    } else {
        return $self->backend->run_ssh_cmd($cmd, %credentials, %args);
    }
```

Comment (okurz) [reply consoles/sshVirtsh.pm:641]: then please use the ternary operator directly:

```suggestion
    return $args{wantarray} ? @{$self->backend->run_ssh_cmd($cmd,  %credentials, %args)} : $self->backend->run_ssh_cmd($cmd, %credentials, %args);
```

**Comment by okurz** [comment]
Please add corresponding unit tests

**Comment by b10n1k** [comment]
(much better permanent solution)[https://github.com/os-autoinst/os-autoinst/pull/1660]

**Pull request closed without merging**